### PR TITLE
fix(node_exporter): correct indent in systemd template

### PR DESCRIPTION
**Pull request by tigattack**
Before:
image

After:
image

### CODE REVIEW

**Comment by SuperQ** [comment]
Looks like this breaks the startup of the `alternative` test. The exporter fails to run.

File: roles/node_exporter/templates/node_exporter.service.j2 [hunk 1]
```diff
@@ -23,13 +23,13 @@ ExecStart={{ node_exporter_binary_install_dir }}/node_exporter \
 {% for collector in node_exporter_disabled_collectors %}
     '--no-collector.{{ collector }}' \
 {% endfor %}
-{% if node_exporter_tls_server_config | length > 0 or node_exporter_http_server_config | length > 0 or node_exporter_basic_auth_users | length > 0 %}
+{% if node_exporter_tls_server_config | length > 0 or node_exporter_http_server_config | length > 0 or node_exporter_basic_auth_users | length > 0 -%}
```
Comment (SuperQ) [inline roles/node_exporter/templates/node_exporter.service.j2:26]: I think this part of the change is breaking the alternative test.

```suggestion
{% if node_exporter_tls_server_config | length > 0 or node_exporter_http_server_config | length > 0 or node_exporter_basic_auth_users | length > 0 %}
```

File: roles/node_exporter/templates/node_exporter.service.j2 [hunk 2]
```diff
@@ -23,13 +23,13 @@ ExecStart={{ node_exporter_binary_install_dir }}/node_exporter \
...
     {% if node_exporter_version is version('1.5.0', '>=') %}
     '--web.config.file=/etc/node_exporter/config.yaml' \
     {% else %}
     '--web.config=/etc/node_exporter/config.yaml' \
     {% endif %}
-{% endif %}
+{% endif -%}
```
Comment (SuperQ) [inline roles/node_exporter/templates/node_exporter.service.j2:32]:
```suggestion
{% endif %}
```

**Review by gardar — CHANGES_REQUESTED** [review]
Please rebase

**Pull request closed without merging**